src/nimblepkg/nimscriptapi.nim

```diff
@@ -233,7 +233,7 @@ template after*(action: untyped, body: untyped): untyped =
     success = true
     retVal = `action After`()
 
-const nimbleExe* {.strdefine.} = "nimble"
+const nimbleBin* {.strdefine.} = "nimble"
```

Comment:
I like `nimbleExe` much better, "bin" is some silly UNIX misnomer. It's a computer, everything is binary.

Reply:
I don't really get "bin," but I changed it to match `nimBin` in `options.nim` https://github.com/nim-lang/nimble/blob/43a9fbffa4a28e7de07883adb09fc4a0bf4133f9/src/nimblepkg/options.nim#L44

Want to change `nimBin` to `nimExe`?